### DISCUSSION TITLE
OAuth2: Header issue

### DISCUSSION THREAD

**Discussion by CungVietDuy** [fastapi/fastapi#6533] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
async def get_current_user(
    token: str = Depends(oauth2_scheme), db: AsyncIOMotorClient = Depends(get_database)
):
    credentials_exception = HTTPException(
        status_code=HTTP_401_UNAUTHORIZED,
        detail="Could not validate credentials",
        headers={"WWW-Authenticate": "Bearer"}
    )
    try:
        payload = jwt.decode(token, SECRET_KEY, algorithms=[ALGORITHM])
        email: str = payload.get("sub")
        if email is None:
            raise credentials_exception
        token_data = TokenPlayload(email=email)
    except JWTError:
        raise credentials_exception

    dbuser = await get_user_by_email(db, token_data.email)
    if not dbuser:
        raise credentials_exception
    user = User(**dbuser.dict(), token=token)
    return user
```

### Description

I follow the documentation for security part. 
The authorization works, however, when I want to use it as dependencies for other routes. I got this error.

### Operating System

Linux

### FastAPI Version

0.68.0

### Python Version

3.8.3

### Additional Context

![Screenshot 2021-08-09 234947](https://user-images.githubusercontent.com/68604305/128726627-f162bc8b-08a0-4cf9-a30b-22bca3747016.png)

![Screenshot 2021-08-09 234638](https://user-images.githubusercontent.com/68604305/128725879-65a28bfc-49e4-414a-bf26-13f979f9a26c.png)

**Comment by CungVietDuy** [comment]
user route

```python
from fastapi import APIRouter, Body, Depends

from ...core.jwt import get_current_user
from ...crud.shortcuts import check_free_username_and_email
from ...crud.user import update_user
from ...db.mongodb import AsyncIOMotorClient, get_database
from ...models.user import User, UserInResponse, UserInUpdate

router = APIRouter()


@router.get("/user", response_model=UserInResponse, tags=["users"])
async def retrieve_current_user(user: User = Depends(get_current_user)):
    return UserInResponse(**user)
```

**Comment by Kludex** [comment]
Can I see the import statement of `HTTPException`?

**Comment by CungVietDuy** [comment]
@Kludex 

```python
from starlette.exceptions import HTTPException
from starlette.status import HTTP_401_UNAUTHORIZED
```

**Comment by CungVietDuy** [comment]
@Kludex thanks for your recommendation. I changed the import statements and it worked.

**Comment by Kludex** [comment, marked as answer]
You're welcome. Would you mind closing the issue? :kissing: 

For reference, FastAPI creates its own `HTTPException`, inheriting from Starlette `HTTPException`:
https://github.com/tiangolo/fastapi/blob/25a48287ed6043f7ee81417cef3c4d1f1dc56f72/fastapi/exceptions.py#L8-L16
